### DISCUSSION TITLE
Install Transmission Daemon as service from source

### DISCUSSION THREAD

**Discussion by tearfur** [transmission/transmission#3008] [Q&A] [answered]
I am on Ubuntu 22.04

I am looking to use Transmission with Tr Web Control. Not wanting to screw up an installation by having APT overwrite some files from Web Control, and not able to find any info regarding this is actually safe, I decided to build and install from the 3.0.0 release.

While I was able to build and install, it was not added as a `systemctl` service. How would I configure the build to do that?

**Comment by maxz** [comment, marked as answer]
This is not part of the build process for Transmission.

You can use the service file which is part of the Transmission source e. g. in `daemon/transmission-daemon.service` or your distribution's service file which comes as part of the `transmission-daemon` package and is found at `/lib/systemd/system/transmission-daemon.service`, according to Ubuntu's [package index](https://packages.ubuntu.com/jammy/amd64/transmission-daemon/filelist).
You would have to adjust the paths accordingly to match your compiled Transmission's location.

You would have to place your own service file in `/etc/systemd/system/`.
